pkg/server/api_v2_databases_metadata_test.go

```diff
@@ -752,31 +754,34 @@ func TestGetTableMetadataUpdateJobStatus(t *testing.T) {
 func TestTriggerMetadataUpdateJob(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
-	skip.UnderStress(t, "too slow under stress")
 	jobCompletedChan := make(chan interface{})
 	jobReadyChan := make(chan interface{})
 	defer close(jobCompletedChan)
 	defer close(jobReadyChan)
-	testCluster := serverutils.StartCluster(t, 3, base.TestClusterArgs{
-		ServerArgs: base.TestServerArgs{
-			Knobs: base.TestingKnobs{
-				TableMetadata: &tablemetadatacache_util.TestingKnobs{
-					OnJobReady: func() {
-						jobReadyChan <- struct{}{}
-					},
-					OnJobComplete: func() {
-						jobCompletedChan <- struct{}{}
-					},
+	ctx := context.Background()
+	var zeroDuration time.Duration
+	ts := serverutils.StartServerOnly(t, base.TestServerArgs{
+		Knobs: base.TestingKnobs{
+			TableMetadata: &tablemetadatacacheutil.TestingKnobs{
+				OnJobReady: func() {
+					jobReadyChan <- struct{}{}
+				},
+				OnJobComplete: func() {
+					jobCompletedChan <- struct{}{}
+				},
+				TableMetadataUpdater: &tablemetadatacacheutil.NoopUpdater{},
```

Comment:
I don't think we need to run / test the actual job here since we already have coverage for that in other tests. All we care about for these tests is the triggering of the job and not the actually updating of the tables.